arewegleanyet.js

```diff
@@ -221,6 +229,26 @@ async function processRelease() {
       }
     }
 
+    let environmentProbes = 0;
+    let environmentMetrics = 0;
+    // The Legacy Telemetry Environment is annoying,
+    // so we list the counts in comments.
+    let legacyCount = cache.environment.match(/Legacy Count: ([0-9]+)/);
+    if (!legacyCount) {
```

Comment:
and this to be a check without the ! and without else block. (same applies to the other variable)